db/raw_doc_with_inline_sync.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
I think I'd rather be more deliberate about switching to the use of embed and standalone json files for testing, instead of just adding this single standalone file - would like to think about organization, packaging, etc.  Any reason this can't be inline with the test file, like we do in other spots?